src/main/java/org/broadinstitute/hellbender/tools/walkers/SplitIntervals.java

```diff
@@ -56,10 +56,10 @@
 public class SplitIntervals extends GATKTool {
 
     public static final String SCATTER_COUNT_SHORT_NAME = "scatter";
-    public static final String SCATTER_COUNT_LONG_NAME = "scatter_count";
+    public static final String SCATTER_COUNT_LONG_NAME = "scatter-count";
```

Comment:
> Update VariantProgramGroup-->IntervalProgramGroup.java

done

> Add "See --subdivision-mode parameter for options.

done

> I feel like there should be a warning someplace about not using intervals that split contigs into consecutive adjacent intervals with assembly callers

I noted that it may affect results.  I'm loathe to make it an outright warning because it's the tool default and we haven't heard of problems with it yet.

Reply:
Ok.